### PR TITLE
🚨 EUDEMO DOC201

### PR DESCRIPTION
**Pull request by je-cook**
## Linked Issues

See #3504 

## Checklist

I confirm that I have completed the following checks:

- [ ] Tests run locally and pass `pytest tests --reactor`
- [ ] Code quality checks run locally and pass `pre-commit run --from-ref develop --to-ref HEAD`
- [ ] Documentation built locally and checked `sphinx-build -W documentation/source documentation/build`

### CODE REVIEW

File: eudemo/eudemo/maintenance/lower_port/duct_designer.py [hunk 1]
```diff
@@ -93,7 +93,20 @@ def __init__(
         self.port_width = self.params.lp_width.value
 
     def run(self) -> tuple[BluemiraFace, BluemiraFace, BluemiraWire, BluemiraWire]:
-        """Run method of Designer"""
+        """
+        Run method of Designer
+
+        Returns
+        -------
+        :
```
Comment (oliverfunk) [inline eudemo/eudemo/maintenance/lower_port/duct_designer.py:101]: Add var names here?

**Review by oliverfunk — APPROVED** [review]

File: eudemo/eudemo/blanket/builder.py [hunk 2]
```diff
@@ -91,6 +101,11 @@ def build_xz(self, ibs_silhouette: BluemiraFace, obs_silhouette: BluemiraFace):
     def build_xy(self, segments: list[PhysicalComponent]):
         """
         Build the x-y components of the blanket.
+
+        Returns
+        -------
+        :
+            the xy components
```
Comment (je-cook) [inline eudemo/eudemo/blanket/builder.py:108]:
```suggestion
            The xy components
```

File: eudemo/eudemo/blanket/builder.py [hunk 3]
```diff
@@ -109,6 +124,11 @@ def build_xy(self, segments: list[PhysicalComponent]):
     def build_xyz(self, segments: list[PhysicalComponent], degree: float = 360.0):
         """
         Build the x-y-z components of the blanket.
+
+        Returns
+        -------
+        :
+            the xyz components
```
Comment (je-cook) [inline eudemo/eudemo/blanket/builder.py:131]:
```suggestion
            The xyz components
```

File: eudemo/eudemo/blanket/builder.py [hunk 4]
```diff
@@ -122,6 +142,11 @@ def build_xyz(self, segments: list[PhysicalComponent], degree: float = 360.0):
     def get_segments(self, ibs_silhouette: BluemiraFace, obs_silhouette: BluemiraFace):
         """
         Create segments of the blanket from inboard and outboard silhouettes
+
+        Returns
+        -------
+        :
+            blanket segments
```
Comment (je-cook) [inline eudemo/eudemo/blanket/builder.py:149]:
```suggestion
            Blanket segments
```

File: eudemo/eudemo/blanket/designer.py [hunk 5]
```diff
@@ -132,7 +138,18 @@ def _remove_gap_and_merge(
...
+        Run the blanket design problem.
+
+        Returns
+        -------
+        cut_ib:
+            Inboard blanket face
+        cut_ob:
+            Outboard blanket face
+        panel_points:
+            the panel points of the blanket
```
Comment (je-cook) [inline eudemo/eudemo/blanket/designer.py:151]:
```suggestion
            The panel points of the blanket
```

File: eudemo/eudemo/blanket/designer.py [hunk 6]
```diff
@@ -178,14 +196,28 @@ def segment_blanket(self) -> BlanketSegments:
...
 
     def _make_cutting_face(self) -> BluemiraFace:
-        """Make a face that can be used to cut the blanket into inboard & outboard."""
+        """
+        Make a face that can be used to cut the blanket into inboard & outboard.
+
+        Returns
+        -------
+        cut_zone:
+            the face cut for the inboard outboard split
```
Comment (je-cook) [inline eudemo/eudemo/blanket/designer.py:219]:
```suggestion
            The face cut for the inboard outboard split
```

File: eudemo/eudemo/blanket/designer.py [hunk 7]
```diff
@@ -201,6 +233,13 @@ def _make_cutting_face(self) -> BluemiraFace:
     def _cut_geom(geom: _GeomT, cut_tool: BluemiraFace) -> tuple[_GeomT, _GeomT]:
         """Cut the given geometry into two using the given cutting tool.
 
+        Returns
+        -------
+        inboard:
+            inboard geometry
```
Comment (je-cook) [inline eudemo/eudemo/blanket/designer.py:239]:
```suggestion
            Inboard geometry
```

File: eudemo/eudemo/blanket/designer.py [hunk 8]
```diff
@@ -201,6 +233,13 @@ def _make_cutting_face(self) -> BluemiraFace:
...
+        outboard:
+            outboard geometry
```
Comment (je-cook) [inline eudemo/eudemo/blanket/designer.py:241]:
```suggestion
            Outboard geometry
```

File: eudemo/eudemo/vacuum_vessel.py [hunk 9]
```diff
@@ -198,6 +214,11 @@ def build_xz(
     def build_xy(self, vv_face: BluemiraFace) -> list[PhysicalComponent]:
         """
         Build the x-y components of the vacuum vessel.
+
+        Returns
+        -------
+        :
+            the xy component parts
```
Comment (je-cook) [inline eudemo/eudemo/vacuum_vessel.py:221]:
```suggestion
            The xy component parts
```

File: eudemo/eudemo/tools.py [hunk 10]
```diff
@@ -21,6 +21,11 @@
 def get_inner_cut_point(breeding_blanket_xz, r_inner_cut):
     """
     Get the inner cut point of the breeding blanket geometry.
+
+    Returns
+    -------
+    :
+        the inner cut point
```
Comment (je-cook) [inline eudemo/eudemo/tools.py:28]:
```suggestion
        The inner cut point
```

File: eudemo/eudemo/tf_coils/tf_coils.py [hunk 11]
```diff
@@ -723,11 +814,16 @@ def _build_xyz_case(
...
+        return casing
 
     def _make_ins_xsec(self) -> tuple[BluemiraFace, BluemiraFace]:
         """
         Make the insulation + insertion gap x-y cross-section faces
+
+        Returns
+        -------
+        :
+            inner and outer cross-section faces
```
Comment (je-cook) [inline eudemo/eudemo/tf_coils/tf_coils.py:826]:
```suggestion
            Inner and outer cross-section faces
```

File: eudemo/eudemo/tf_coils/tf_coils.py [hunk 12]
```diff
@@ -677,17 +763,22 @@ def _build_xyz_ins(
...
         outer_xs: BluemiraWire,
-    ) -> tuple[BluemiraSolid, PhysicalComponent]:
+    ) -> PhysicalComponent:
         """
         Casing x-y-z
+
+        Returns
+        -------
+        :
+            the casing component
```
Comment (je-cook) [inline eudemo/eudemo/tf_coils/tf_coils.py:781]:
```suggestion
            The casing component
```

File: eudemo/eudemo/tf_coils/tf_coils.py [hunk 13]
```diff
@@ -545,6 +601,11 @@ def build_xyz(
     def _build_xz_wp(self) -> tuple[BluemiraWire, BluemiraWire, PhysicalComponent]:
         """
         Winding pack x-z
+
+        Returns
+        -------
+        :
+            the xz winding pack inner and outer wires and component
```
Comment (je-cook) [inline eudemo/eudemo/tf_coils/tf_coils.py:608]:
```suggestion
            The xz winding pack inner and outer wires and component
```

File: eudemo/eudemo/tf_coils/tf_coils.py [hunk 14]
```diff
@@ -395,6 +426,11 @@ def mock(self) -> tuple[GeometryParameterisation, BluemiraWire]:
         """
         Mock a design of TF coils using the original parameterisation of the current
         centreline.
+
+        Returns
+        -------
+        :
+            the parameterisation and the winding pack cross section
```
Comment (je-cook) [inline eudemo/eudemo/tf_coils/tf_coils.py:433]:
```suggestion
            The parameterisation and the winding pack cross section
```

File: eudemo/eudemo/tf_coils/tf_coils.py [hunk 15]
```diff
@@ -374,6 +400,11 @@ def read(self) -> tuple[GeometryParameterisation, BluemiraWire]:
         Read in a file to set up a specified GeometryParameterisation and extract the
         current centreline.
 
+        Returns
+        -------
+        :
+            the parameterisation and the winding pack cross section
```
Comment (je-cook) [inline eudemo/eudemo/tf_coils/tf_coils.py:406]:
```suggestion
            The parameterisation and the winding pack cross section
```

File: eudemo/eudemo/tf_coils/tf_coils.py [hunk 16]
```diff
@@ -301,6 +322,11 @@ def run(self) -> tuple[GeometryParameterisation, BluemiraWire]:
         Run the specified design optimisation problem to generate the TF coil winding
         pack current centreline.
 
+        Returns
+        -------
+        :
+            the parameterisation and the winding pack cross section
```
Comment (je-cook) [inline eudemo/eudemo/tf_coils/tf_coils.py:328]:
```suggestion
            The parameterisation and the winding pack cross section
```

File: eudemo/eudemo/tf_coils/tf_coils.py [hunk 17]
```diff
@@ -234,6 +250,11 @@ def _make_wp_xs(self, inboard_centroid: float) -> BluemiraWire:
     def _make_centreline_koz(self, keep_out_zone: BluemiraWire) -> BluemiraWire:
         """
         Make a keep-out-zone for the TF coil centreline optimisation problem.
+
+        Returns
+        -------
+        :
+            keep-out-zone wire
```
Comment (je-cook) [inline eudemo/eudemo/tf_coils/tf_coils.py:257]:
```suggestion
            keep-out-zone wire
```
```suggestion
            Keep-out-zone wire
```

File: eudemo/eudemo/tf_coils/tf_coils.py [hunk 18]
```diff
@@ -97,7 +103,12 @@ def xz_outer_boundary(self) -> BluemiraWire:
         )
 
     def xz_face(self) -> BluemiraFace:
-        """Return the x-z face of the TF Coils."""
+        """
+        Returns
+        -------
+        :
+            the x-z face of the TF Coils.
```
Comment (je-cook) [inline eudemo/eudemo/tf_coils/tf_coils.py:110]:
```suggestion
            The x-z face of the TF Coils.
```

File: eudemo/eudemo/tf_coils/tf_coils.py [hunk 19]
```diff
@@ -82,12 +82,18 @@ def field(
...
         """
         return self._field_solver.field(x, y, z)
 
     def xz_outer_boundary(self) -> BluemiraWire:
-        """Return the outer xz-boundary of the TF Coils."""
+        """
+        Returns
+        -------
+        :
+            the outer xz-boundary of the TF Coils.
```
Comment (je-cook) [inline eudemo/eudemo/tf_coils/tf_coils.py:95]:
```suggestion
            The outer xz-boundary of the TF Coils.
```

File: eudemo/eudemo/reactor.py [hunk 20]
```diff
@@ -315,8 +388,14 @@ def build_lower_port(
...
     cryostat_xz_boundary,
-):
-    """Builder for the Lower Port and Duct"""
+) -> tuple[Component, ...]:
+    """Builder for the Lower Port and Duct
+
+    Returns
+    -------
+    :
+        lower port components
```
Comment (je-cook) [inline eudemo/eudemo/reactor.py:397]:
```suggestion
        Lower port components
```

File: eudemo/eudemo/reactor.py [hunk 21]
```diff
@@ -298,9 +364,16 @@ def build_upper_port(
...
+def build_equatorial_port(
+    params, build_config, cryostat_ts_xz_boundary
+) -> tuple[Component, ...]:
     """
     Build the equatorial port for the reactor.
+
+    Returns
+    -------
+    :
+        equatorial port components
```
Comment (je-cook) [inline eudemo/eudemo/reactor.py:376]:
```suggestion
        Equatorial port components
```

File: eudemo/eudemo/reactor.py [hunk 22]
```diff
@@ -287,9 +348,14 @@ def build_upper_port(
...
     cryostat_ts_xz_boundary: BluemiraFace,
-):
+) -> tuple[Component, ...]:
     """
     Build the upper port for the reactor.
+
+    Returns
+    -------
+    :
+        upper port components
```
Comment (je-cook) [inline eudemo/eudemo/reactor.py:358]:
```suggestion
        Upper port components
```

**Pull request merged**